Resources/Prototypes/_SpaceCats/Entities/Objects/Misc/refill_cartridge.yml

```
@@ -0,0 +1,14 @@
+# Оригинал данного файла был сделан @temporaldarkness (discord). Прототип был взят с https://github.com/ss14-ganimed/ENT14-Master.- type: entity
+  id: BookPrinterInkCartridge
```

yamllint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)